README.md

```diff
@@ -27,90 +27,53 @@ Convenience commands have been added to `fabfile.py` to help you interact with t
 Once installed, you can type `fab -l` to see a list of available commands.
 
 
-### 1. Build Docker containers
+### 1. Build and start Docker containers
 
-```console
-$ fab build
-```
-
-### 2. Start Docker containers
-
-```console
-$ fab start
-```
-
-### 3. Start a shell session with the 'web' container
-
-```console
-$ fab sh
-```
-
-### 5. Apply database migrations
-
-```console
-$ python manage.py migrate
+```sh
+fab start
```

Comment:
```suggestion
fab build && fab start
```

Does it need to be this?

Reply:
It shouldn't need to be.

`build` just builds the Docker images using `docker-compose build`.

`start` aliases `docker-compose up -d` which "Builds, (re)creates, starts, and attaches to containers for a service." (https://docs.docker.com/engine/reference/commandline/compose_up/#description)

I didn't know if there was a use case for ever just building but not running, but maybe I've made an assumption?

Reply:
> I didn't know if there was a use case for ever just building but not running, but maybe I've made an assumption

Good point, for me, I've always used both of them in sequence, if any other developer has a case, they should post here.

Reply:
Cool, happy with that! 😃 I always just run `fab start` on its own